roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/clusterrole-ingress-nginx.yml.j2

```diff
@@ -16,15 +16,16 @@ rules:
   - apiGroups: [""]
     resources: ["services"]
     verbs: ["get", "list", "watch"]
-  - apiGroups: ["extensions","networking.k8s.io"]
-    resources: ["ingresses", "ingressclasses"]
+  - apiGroups: ["networking.k8s.io"]
+    resources: ["ingresses"]
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["create", "patch"]
   - apiGroups: ["extensions","networking.k8s.io"]
     resources: ["ingresses/status"]
     verbs: ["update"]
-  - apiGroups: [networking.k8s.io"]
+  - apiGroups: ["networking.k8s.io"]
```

Comment:
The list of verbs and apiGroups is the same as the one above. It looks cleaner to just group resources in the entry above and delete this typo'ed line.

Reply:
That is a nice point.
I agree with @cristicalin here

Reply:
@cristicalin @oomichi 
nope, i do not agree: the one above is only for `ingress/status` and update.
I recommend to not reinvent the wheel and stay close to https://github.com/kubernetes/ingress-nginx/blob/main/deploy/static/provider/baremetal/1.21/deploy.yaml#L209 for easier merge in the future.

Reply:
Thank you for your explanation, that makes sense for easy maintenance.

/ok-to-test
/lgtm